mesheryctl/internal/cli/root/components/list.go

```diff
@@ -134,7 +144,51 @@ mesheryctl exp components list --page 2
 	},
 }
 
+func displayCount() error {
+	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+	if err != nil {
+		return err
+	}
+
+	baseUrl := mctlCfg.GetBaseMesheryURL()
+
+	// Since we are not searching for particular component, we don't need to pass any search parameter
+	url := fmt.Sprintf("%s/api/meshmodels/components?pagesize=all", baseUrl)
```

Comment:
If you use pagesize=1 instead of fetching all results, the output will be much faster since you only need the count. Currently, it takes 5-6 seconds to fetch the result. @MUzairS15 , should we implement this optimization to reduce the time?

Reply:
@Jougan-0 thks fo rpointing out htis.
I saw it takes sometimes during the test but just moved the code as is since thoughts it was already tested. 
If we have same result and a better response time I think we should do it.

Reply:
Definitely we should go this way, gonna push the changes

Benchmark 

before `pagesize=all`
```bash
Total components present:  4453

real	        0m17.771s
user	0m1.330s
sys	        0m0.248s
```

after `pagesize=1`
```bash
Total components present:  4453

real	        0m0.242s
user	0m0.168s
sys	        0m0.065s
```

Reply:
done